### DISCUSSION TITLE
CD Burner

### DISCUSSION THREAD

**Discussion by RaptaG** [strawberrymusicplayer/strawberry#1282] [Ideas]
While I know that this has been been [purposely excluded from the upstream of Clementine](https://wiki.strawberrymusicplayer.org/wiki/Differences_from_Clementine#Features_missing), I believe it is a very useful feature for Strawberry. I mean, it can already write onto other device types, why not one of the most common ones?

(P.S: Once I learn more C++ and this has not been implemented, I'll open up a PR)

**Comment by jonaski** [comment]
Add this to https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
